src/components/Upload/FileUploadIcon.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from 'react';
+import { withTheme } from 'styled-components';
+import { ComponentTheme } from '../../theme/index';
+
+type Props = {
+  color?: string;
```

Comment:
Instead of accepting whatever color string you want, it might be better to work with color-"variants".
`color?: 'primary' | 'secondary' | 'warning' | 'tertiary'`

and that would mean you could use something like this
`fill={color ? theme.upload.svgIcon.fill[color] : '#0093B2'`

and in your story it would be
`color="primary"`